pkg/minikube/config/config.go

```diff
@@ -56,6 +56,8 @@ const (
 	AddonImages = "addon-images"
 	// AddonRegistries stores custom addon images config
 	AddonRegistries = "addon-registries"
+	// AddonList represents the key for addons parameter
+	AddonList = "addons"
```

Comment:
lets change this to something obvious that it is the flag name how about "AddonListFlag"

Reply:
Yeah, that makes sense. Making the changes, thanks!

Reply:
Done @medyagh :smile: